examples/package.json

```diff
@@ -1,31 +1,32 @@
     "test": "npm run lint && npm run types",
-    "clean-examples": "rm -rf public .parcel-cache",
-    "start:map": "npm run clean-examples && PARCEL_AUTOINSTALL=false parcel serve ./basic-google-map/index.html --dist-dir public --port 1234",
-    "start:map-with-markers": "npm run clean-examples && PARCEL_AUTOINSTALL=false parcel serve ./google-map-with-markers/index.html --dist-dir public --port 1234",
-    "start:geocoding": "npm run clean-examples && PARCEL_AUTOINSTALL=false parcel serve ./geocoding/index.html --dist-dir public --port 1234",
-    "start:places": "npm run clean-examples && PARCEL_AUTOINSTALL=false parcel serve ./places/index.html --dist-dir public --port 1234",
-    "start:places-autocomplete": "npm run clean-examples && PARCEL_AUTOINSTALL=false parcel serve ./places-autocomplete/index.html --dist-dir public --port 1234",
-    "start:directions": "npm run clean-examples && PARCEL_AUTOINSTALL=false parcel serve ./directions/index.html --dist-dir public --port 1234",
-    "start:distance-matrix": "npm run clean-examples && PARCEL_AUTOINSTALL=false parcel serve ./distance-matrix/index.html --dist-dir public --port 1234"
+    "clean-examples": "rm -rf ./public ../.parcel-cache",
+    "start:map": "npm run clean-examples && cross-env $(cat .env) PARCEL_AUTOINSTALL=false parcel serve ./basic-google-map/index.html --dist-dir public --port 1234 --no-cache",
+    "start:map-with-markers": "npm run clean-examples && cross-env $(cat .env) PARCEL_AUTOINSTALL=false parcel serve ./google-map-with-markers/index.html --dist-dir public --port 1234 --no-cache",
+    "start:geocoding": "npm run clean-examples && cross-env $(cat .env) PARCEL_AUTOINSTALL=false parcel serve ./geocoding/index.html --dist-dir public --port 1234 --no-cache",
+    "start:places": "npm run clean-examples && cross-env $(cat .env) PARCEL_AUTOINSTALL=false parcel serve ./places/index.html --dist-dir public --port 1234 --no-cache",
+    "start:places-autocomplete": "npm run clean-examples && cross-env $(cat .env) PARCEL_AUTOINSTALL=false parcel serve ./places-autocomplete/index.html --dist-dir public --port 1234 --no-cache",
+    "start:directions": "npm run clean-examples && cross-env $(cat .env) PARCEL_AUTOINSTALL=false parcel serve ./directions/index.html --dist-dir public --port 1234 --no-cache",
+    "start:distance-matrix": "npm run clean-examples && cross-env $(cat .env) PARCEL_AUTOINSTALL=false parcel serve ./distance-matrix/index.html --dist-dir public --port 1234 --no-cache",
+    "preversion": "echo \"To create a new library version run 'npm version -w library' in the repository root.\""
   },
   "license": "MIT",
   "dependencies": {
-    "@ubilabs/google-maps-react-hooks": "^1.6.2",
+    "@ubilabs/google-maps-react-hooks": "*",
     "react": "^18.2.0",
     "react-dom": "^18.2.0"
   },
   "devDependencies": {
     "@types/googlemaps": "^3.43.3",
     "@types/react": "^18.0.21",
     "@types/react-dom": "^18.0.6",
+    "cross-env": "^7.0.3",
```

Comment:
What does `cross-env` do here?

Reply:
It's needed because we need the parcel project root to be in the root of the repository for the watcher to detect changes in the library, but the .env file is in the examples folder and we would like to keep it there, because it belongs to the examples. Since parcel looks for the .env files in the project root and there's no option to change this, I implemented cross-env to load the .env from the examples folder.
Other solutions are welcome.

Reply:
Works well now with the `.env` in the examples now. Thank for adjusting. I would keep it like this, if there are no other suggestions here :)

Reply:
One could define each example as a workspace and run parcel in its own workspace context. But I doubt that this would make things easier. :)

Reply:
Then we would have the watcher problem again. If you run parcel in a workspace, the watcher doesn't detect changes in other workspaces (in this case library and examples).